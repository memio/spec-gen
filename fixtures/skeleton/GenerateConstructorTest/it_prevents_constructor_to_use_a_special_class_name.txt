<?php

namespace Vendor\Project;

class MyClass
{
}
