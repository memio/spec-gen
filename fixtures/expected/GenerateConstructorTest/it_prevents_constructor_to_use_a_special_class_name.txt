<?php

namespace Vendor\Project;

class MyClass
{
    private $argument1;

    private $argument2;

    private $argument3;

    private $argument4;

    public function __construct(
        int $argument1,
        float $argument2,
        string $argument3,
        bool $argument4
    ) {
        $this->argument1 = $argument1;
        $this->argument2 = $argument2;
        $this->argument3 = $argument3;
        $this->argument4 = $argument4;
    }
}
